Add \iff, \implies, \impliedby support

edemaine opened the pull request:
Fix #190 via @ronkok's code.

I thought about tests but couldn't think of anything that interesting.  A `expect(...).toParseLike` doesn't seem to check anything.  I could add a screenshot test, but it is also pretty "boring".

ronkok commented:
That looks good. And kudos to @gagern, who made macros possible.

kevinbarabash commented on src/macros.js:
```diff
@@ -21,3 +21,10 @@ defineMacro("\\endgroup", "}");
+
+// \def\iff{\DOTSB\;\Longleftrightarrow\;}
+// \def\implies{\DOTSB\;\Longrightarrow\;}
+// \def\impliedby{\DOTSB\;\Longleftarrow\;}
```
Can you add a TODO above these to indicate that this is what we want the the implementation of these commands to eventually look like?

edemaine replied:
Added.

edemaine commented:
@kevinbarabash Implemented both your suggestions (and rebased).  Thanks for the feedback!

I also noticed that, while @ronkok's code had a `\mathrel` surrounding the operators, the `amstex.sty` code did not, so I switched to the latter.

ronkok commented:
I'm good with the change. The span still is coded as a `class="mrel"`.

kevinbarabash commented:
@edemaine thanks for adding these new commands.
